moabb/evaluations/evaluations.py

```diff
@@ -18,23 +18,31 @@ class TrainTestEvaluation(BaseEvaluation):
     (not compatible with multi-task/transfer/multi-dataset schemes)
     '''
 
-    def extract_data_from_cont(self, ep_list, event_id):
-        event_epochs = {key: [] for key in event_id.keys()}
-        for epoch in ep_list:
+    def extract_data_from_cont(self, band_dict, event_id):
```

Comment:
extract_data_from_cont now takes  a dictionary with the epochs from each band (if dictionary length is one, assumes one band) and returns either a 3-D or 4-D feature matrix depending on the case. It is terribly hacky but I don't know how else to fix this yet